cl/api/utils.py

```diff
@@ -584,6 +585,44 @@ def get_next_webhook_retry_date(retry_counter: int) -> datetime:
     return new_next_retry_date
 
 
+WEBHOOK_MAX_RETRY_COUNTER = 7
+
+
+def disabling_webhook_check(webhook_event: WebhookEvent) -> None:
```

Comment:
This name is a bit funky. How about:

```suggestion
def check_webhook_failure_count_and_notify(webhook_event: WebhookEvent) -> None:
```

Reply:
Changed.